src/qt/pivx/sendcustomfeedialog.cpp

```diff
@@ -126,6 +126,9 @@ void SendCustomFeeDialog::accept()
         inform(tr("Fee too high. Must be below: %1").arg(
                 BitcoinUnits::formatWithUnit(walletModel->getOptionsModel()->getDisplayUnit(), insaneFee)));
     } else if (customFee < CWallet::GetRequiredFee(1000)) {
+        CAmount nFee;
+        walletModel->getWalletCustomFee(nFee);
```

Comment:
Would be good to check for the `getWalletCustomFee` return boolean value or use the `hasWalletCustomFee` method before getting the wallet custom fee, so in case of having the wallet custom fee disabled (`CWallet::fUseCustomFee`), the GUI use the minimum required fee and not the stored value.

In other words, something like:
```
CAmount nFee = 0;
if (walletModel->hasWalletCustomFee()) {
     walletModel->getWalletCustomFee(nFee);
} else {
     nFee = CWallet::GetRequiredFee(1000);
}
```

Side topic, a future good work on this area would be to encapsulate every `CWallet` related function call into the wallet model. None of the widgets should have access to the backend related files/classes, only connect to the models.

Reply:
Maybe we can merge this PR, and fix this thing in a quick follow-up PR?

As for the side topic... check #2293 :)

Reply:
Cool for #2293 👌 .
Ok, np. let's merge this one and will do a quick follow-up PR to fix the problem.

Guess that @MishaPozdnikin isn't around anymore.